### PR TITLE
Mobile Menu

### PR DESCRIPTION
**Pull request by DarkPurple141**
After reverting previous direct dev commits.

screen shot 2019-02-11 at 7 46 26 pm

### CODE REVIEW

**Comment by NunoDasNeves** [comment]
Hmm, it looks like the logout button isn't quite right on my Firefox devtools phone-thingy:
screenshot from 2019-02-11 20-07-21

**Comment by DarkPurple141** [comment]
> Hmm, it looks like the logout button isn't quite right on my Firefox devtools phone-thingy:
> screenshot from 2019-02-11 20-07-21

Hmm. Sec

File: frontend/src/components/Nav/MiniMenu.vue [hunk 1]
```diff
@@ -20,15 +20,23 @@
           class="menu-item"
         >{{ item.text }}</router-link>
       </div>
-      <slot/>
+      <div
```
Comment (NunoDasNeves) [inline frontend/src/components/Nav/MiniMenu.vue]: Looks great but shouldn't it be a button or h3 or something with `cursor: pointer;` ? I guess this doesn't matter much given it's for mobile

Comment (DarkPurple141) [reply frontend/src/components/Nav/MiniMenu.vue]: yeah because it's mobile doesn't matter.

**Pull request merged**